qualtran/bloqs/qubitization_walk_operator.py

```diff
@@ -64,6 +67,7 @@ class QubitizationWalkOperator(GateWithRegisters):
 
     select: SelectOracle
     prepare: PrepareOracle
+    sum_of_ham_coeffs: Optional[float]
```

Comment:
The default value here should be 1 or something else?

Reply:
I didn't set a default to avoid accidentally forgetting it while building walk ops

But as the `sum_of_ham_coeffs` attribute is not needed in the decomposition/call-graph, it won't break anything. It's only needed for Hamiltonian simulation, in which case it needs to be correctly calculated.

Reply:
right, I think Optional[float] = None in that case.

Reply:
fixed

Reply:
would it be better to not have a default, and let the user explicitly specify `None` when they don't know the value? (just to make sure it's not accidentally omitted)